### ISSUE TITLE
I'm trying to create a module 

### ISSUE THREAD

**Issue by xshadowbringer** [nuclearsilo583/zephyrus-store-preview-new-syntax#154]
Hello, i want to create a module for this shop i don't fully understand how to do it can I share you the source code to inspire me?

**Comment by xshadowbringer** [comment]
And in Items.txt, something like this 

"BuyVip"
	{
		"vip1"
		{
			"name"					"vip1"
			"type"					"buyvipcore"
			"unique_id"				"uid_buyvipcore"
			"Plans"
			{
				"7 days"
				{
					"price" "700000"
					"time" "604800"
				}
			}
		}
	}

**Comment by originalaidn** [comment]
Hi, please test my PR.
In that your items.txt need to seems like this:

"VIP - 1 week"
{
	"type" "buyvip"
	"group" "vip1"
	"time" "604800"
	"price"	"700000"
}

**Comment by xshadowbringer** [comment]
> Hi, please test my PR. In your case the items.txt should look like this:
> 
> "VIP - 1 week" { "type" "buyvip-r1ko" "group" "vip1" "time" "604800" "price" "700000" }

in the VIP_Remove() function i think before "return 0;" should be "VIP_RemoveClientVIP2(0, client, false, true);
	PrintToChat(client, "~# VipSHOP$ Your Vip status was unequipped");"   because if the client sold, traded the item the vip status won't get removed. also in VIP_Equip() the player should get vip status based on the remaining time of the item if its not removed from store

**Comment by originalaidn** [comment]
As far as i remember, you can't sell/trade the item, as it disappears from your inventory as soon as you equip it.

In VIP_Equip() i don't know what you want, the duration is based on the "time" which is in the CFG, u can add for 2 weeks, 1 month, 6 months, etc.. just need to edit the "time". or u want same as u send, in Plans section?

**Comment by xshadowbringer** [comment]
> As far as i remember, you can't sell/trade the item, as it disappears from your inventory as soon as you equip it. --> Store_RemoveItem(client, id);
> 
> In VIP_Equip() i don't know what you want, the duration is based on the "time" which is in the CFG, u can add for 2 weeks, 1 month, 6 months, etc.. just need to edit the "time". or u want same as u send, in Plans section?

Yes, (As far as i remember, you can't sell/trade the item, as it disappears from your inventory as soon as you equip it. -->) this is true but if you equip it and immediately trade it you can the vip and also trade it to another person maybe better to block the trade for this item entirely

**Comment by originalaidn** [comment]
Ahh I see, just add "trade" "0" to the config, but I'll check what you wrote

**Comment by azalty** [comment]
If it's possible to trade an item after using it, then it is a pretty serious exploit indeed. Thanks for notifying us.

**Comment by adyshark** [comment]
store plugin already haves "ignore_vip" option, but works only with flags, can this be implemented with this vip core plugin?

**Comment by adyshark** [comment]
also,after vip expires, in database vip  players  must be removed, to cleanup table.
